interp/runt.toml

```diff
@@ -66,6 +66,42 @@ cmd = """
 """
 timeout = 3
 
+[[tests]]
+name = "invoke"
```

Comment:
Why do these test suites need to be separate from the normal `control` test suite?

Reply:
The current control tests don't take data but the invoke ones do (generally) so the command is different